portfolio/src/main/webapp/art-gallery.js

```diff
@@ -24,33 +24,36 @@ let getLatestOpenedImgIndex;
 let frames = document.querySelectorAll(".art-container");
 
 function init() {
-    if (frames) {
-        frames.forEach(function(frame, index) {
-            frame.onclick = function() {
-            let imageName = frame.children[0].src.split("images/fine-art/thumbnails/")[1];
-            getLatestOpenedImgIndex = imagesArray.indexOf(imageName);
-            let container = document.body;
-            let newImageWindow = document.createElement("div");
-            container.appendChild(newImageWindow);
-            newImageWindow.setAttribute("class", "img-window");
-            newImageWindow.setAttribute("onclick", "closeImg()");
+  if (frames) {
+    frames.forEach(function(frame, index) {
+      //open up larger image when clicked
+      frame.onclick = function() {
+        let imageName = frame.children[0].src.split(
+          "images/fine-art/thumbnails/"
+        )[1];
+        getLatestOpenedImgIndex = imagesArray.indexOf(imageName);
```

Comment:
Should have caught this earlier, but a name like `getLatestOpenedImgIndex` implies this is a function to be called. Could you rename this to `latestOpenedImgIndex`?